src/Mvc/Mvc.ViewFeatures/src/RazorComponents/HtmlRenderer.cs

```diff
@@ -188,6 +194,34 @@ private int RenderFrames(HtmlRenderingContext context, ArrayRange<RenderTreeFram
         }
     }
 
+    private static int RenderTextareaElement(
+         HtmlRenderingContext context,
+         ArrayRange<RenderTreeFrame> frames,
+         int position)
+    {
+        ref var frame = ref frames.Array[position];
+        var result = context.HtmlContentBuilder;
+
+        result.AppendHtml("<");
+        result.AppendHtml(frame.ElementName);
```

Comment:
Minor optimization - since we know ElementName is `textarea` (and lower/uppercase doesn't matter), this could be reduced to:

```suggestion
        result.AppendHtml("<textarea");
```